onnxruntime/core/session/inference_session.cc

```diff
@@ -1550,6 +1550,16 @@ common::Status InferenceSession::Initialize() {
     }
   }
 
+#if !defined(ORT_MINIMAL_BUILD)
```

Comment:
most of the initialization logic happens above in the try block above. 
is there a reason to put this outside, or could it go here?
https://github.com/microsoft/onnxruntime/blob/69e3ab119566f065f8768532f018fdd12bf3a805/onnxruntime/core/session/inference_session.cc#L1514

Reply:
Great catch! Thanks! This portion of code need to be moved upward.